### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by pritipath**
Issue Number: (Link to Github Issue or Azure Dev Ops Task/Story)

## Summary
Update documentation to clarify how to run the sample console app

## Changes

- List of comprehensive changes

## Checklist

- [ ] Local Tests Passing?
- [ ] CICD and Pipeline Tests Passing?
- [ ] Added any new Tests?
- [ ] Documentation Updates Made?
- [ ] Are there any API Changes? If yes, please describe below.
- [ ] This is not a breaking change. If it is, please describe it below.

## Are there API Changes?
If yes, what are the expected API Changes? Please link to an API-Comparison workflow with the API Diff.

## Is this a breaking change?
If yes, what workflow does this break? 

## Anything else?
Other comments, collaborators, etc.
> Please follow [GitHub's suggested syntax](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) to link Pull Requests to Issues via keywords

This PR Closes #<issue_number>

### CODE REVIEW

**Review by gfmatthews — COMMENTED** [review]
I think this clears up what to do but I don't think this doc is living in the right place. If you're just exploring the repo, you'll land in the samples folder but not necessarily have a link to go back and look at this markdown file which tells you how to run things. Some solutions I can think of:

- Move this doc to the samples folder alongside the console app. This breaks the centralization of docs but solves discoverability of this important info.
- Put a small "README" in the sample folder alongside the console app and use that README to point back to this doc. This potentially creates some duplication but solves the discoverability of this document and keeps the centralization of the docs. 

I'm leaning towards option 2 but I'm open to suggestions. @juzuluag / @pritipath what do you all think?

**Comment by juzuluag** [comment]
> I think this clears up what to do but I don't think this doc is living in the right place. If you're just exploring the repo, you'll land in the samples folder but not necessarily have a link to go back and look at this markdown file which tells you how to run things. Some solutions I can think of:
> 
> * Move this doc to the samples folder alongside the console app. This breaks the centralization of docs but solves discoverability of this important info.
> * Put a small "README" in the sample folder alongside the console app and use that README to point back to this doc. This potentially creates some duplication but solves the discoverability of this document and keeps the centralization of the docs.
> 
> I'm leaning towards option 2 but I'm open to suggestions. @juzuluag / @pritipath what do you all think?

I like Option 2.

**Review by gfmatthews — APPROVED** [review]
Looks good!

**Review by JenMadiedo — APPROVED** [review]
Thanks for adding this stuff!

**Comment by pritipath** [comment]
@vaughanknight @Willmish This PR is ready for GSF review

**Review by Kpopovs1 — APPROVED** [review]

**Comment by JenMadiedo** [comment]
> > I think this clears up what to do but I don't think this doc is living in the right place. If you're just exploring the repo, you'll land in the samples folder but not necessarily have a link to go back and look at this markdown file which tells you how to run things. Some solutions I can think of:
> > 
> > * Move this doc to the samples folder alongside the console app. This breaks the centralization of docs but solves discoverability of this important info.
> > * Put a small "README" in the sample folder alongside the console app and use that README to point back to this doc. This potentially creates some duplication but solves the discoverability of this document and keeps the centralization of the docs.
> > 
> > I'm leaning towards option 2 but I'm open to suggestions. @juzuluag / @pritipath what do you all think?
> 
> I like Option 2.

I also like Option 2. I will also add that most the samples have their own README, which is okay as long as the readme is unique to running the sample only and nothing with SDK docs. So I think 2 fits that paradigm perfectly

**Review by Willmish — CHANGES_REQUESTED** [review]
Hi all, I really like option 2 as well, especially since other samples have short READMEs supplied with them as well. Only worry is we will get divergent documentation, and end up with issues like these: https://github.com/Green-Software-Foundation/carbon-aware-sdk/issues/259 , however in this case it seems to be handled nicely by pointing back to https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/docs/packaging.md

Only ask is regarding the [packaging.md](https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/docs/packaging.md#sdk-configuration) doc file, as I just read through it. As this is another doc where `DataSource` configuration is being mentioned, could you add a hyperlink/reference back to [`configuration.md`](https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/docs/configuration.md) file in the [**SDK Configuration** section of the packaging.md file](https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/docs/packaging.md#sdk-configuration)?

(This would be a *fail-safe* documentation, in case the configuration format changes again in the future, `docs/configuration.md` most likely will get updated while `docs/packaging.md` might be omitted)

After this small addition, happy to approve!

File: docs/packaging.md [hunk 1]
```diff
@@ -72,9 +72,10 @@ Below is the environment variable (i.e. bash shell) needed to set up the **Json*
 
 ## Console App Sample
```
[the remark this answers is not in the document]
Comment (pritipath) [reply docs/packaging.md:73]: Thant makes sense. Added reference to configuration.md for clarity.

**Review by Willmish — APPROVED** [review]
LGTM, thanks for adding that link 💯

**Pull request merged**